### DISCUSSION TITLE
[A/L] (2021/22) Induzione - Esercizio 17

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#317] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152323156-3a5335e3-33d8-4407-ad79-6fdc4ec3878b.png)

**Comment by Exyss** [comment]
Soluzione simile all'Esercizio 6 (#107)

![Esercizio 17](https://user-images.githubusercontent.com/65043207/152660444-f7dda52e-2e80-4db7-b9f0-fa3408d5d718.jpg)

**Comment by danieldinella** [comment]
> Soluzione simile all'Esercizio 6 (#107)
> 
> ![Esercizio 17](https://user-images.githubusercontent.com/65043207/152660444-f7dda52e-2e80-4db7-b9f0-fa3408d5d718.jpg)
> 

Attenzione, 0 non è un numero positivo. Dunque il caso base è 1 * 2 * 3.

**Comment by Exyss** [comment]
> > Soluzione simile all'Esercizio 6 (#107)
> > ![Esercizio 17](https://user-images.githubusercontent.com/65043207/152660444-f7dda52e-2e80-4db7-b9f0-fa3408d5d718.jpg)
> 
> Attenzione, 0 non è un numero positivo. Dunque il caso base è 1 * 2 * 3.

Hai perfettamente ragione, andrebbe interpretato come "per ogni N >= 1"
